### PR TITLE
docs(en): add selection sort

### PR DESCRIPTION
**Pull request by Sunainacode**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Added documentation for Selection Sort Algorithm
**Other information:**

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @Sunainacode,
Please rewrite the steps and example sections, take the [Bubble Sort](https://github.com/MakeContributions/DSA/blob/main/docs/en/Sorting/Bubble-Sort.md) as an example

The example section could be like this
```
Given array is
**13, 44, 5, 34, 3**

Sorted array is
**3, 5, 13, 34, 44**

**First pass**

Unsorted array: [13, 44, 5, 34, 3]

Minimum in an unsorted array: 3

Sorted array: [3]

**Second pass**

Unsorted array: [13, 44, 5, 34]

Minimum in unsorted array: 5

Sorted array: [3, 5]

**Third pass**

Unsorted array: [13, 44, 34]

Minimum in an unsorted array: 13

Sorted array: [3, 5, 13]

...
```

**Comment by Sunainacode** [comment]
Hi @ming-tsai.
Can you please review the changes now?

File: docs/en/Sorting/Selection-Sort.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Implementation
+
+- [Java](../../../algorithms/Java/sorting/selection-sort.java)
+- [C++](../../../algorithms/CPlusPlus/Sorting/selection-sort.cpp)
+- [JavaScript](../../../algorithms/JavaScript/src/sorting/selection-sort.js)
+- [Python](../../../algorithms/Python/sorting/selection_sort.py)
+
+## Video URL
+
+<a href="https://www.youtube.com/watch?v=GUDLRan2DWM&list=PL2_aWCzGMAwKedT2KfDMB9YA5DgASZb3U&index=2">Selection Sort</a>
```
Comment (ming-tsai) [inline docs/en/Sorting/Selection-Sort.md:78]: The link would be better using the markdown syntaxis
```suggestion
[Selection Sort](https://www.youtube.com/watch?v=GUDLRan2DWM&list=PL2_aWCzGMAwKedT2KfDMB9YA5DgASZb3U&index=2)
```

**Review by ming-tsai — APPROVED** [review]
Nice work @Sunainacode

**Pull request merged**